src/cpp/server/ServerPAMAuthOverlay.hpp

```diff
@@ -23,7 +23,7 @@ namespace server {
 namespace pam_auth {
 namespace overlay {
 
-void intialize();
+void initialize();
```

Comment:
This change is a little surprising / concerning; the name here is mis-spelled, but also does not match the name here:

https://github.com/rstudio/rstudio/blob/b668ec9ab8d0c2a98c1c1d5c6e85dd541184a7a1/src/cpp/server/ServerPAMAuthOverlay.cpp#L41-L44

This would seem to imply we're not actually calling this method as expected?

Reply:
@kevinushey indeed. Is the plan to open a separate ticket for this?

Reply:
I've done so on our Pro repository.